docs/_layouts/default.html

```diff
@@ -12,9 +12,9 @@
 
     {% include nav-home.html %}
 
-    <div id="content">
+    <main id="content" role="main">
```

Comment:
I always double the <code><main></code> element with the aria role for IE11 who doesn't take care about this element